docs/docsite/rst/dev_guide/developing_python3.rst

```diff
@@ -274,67 +374,42 @@ coded to expect bytes on Python-2 and text on Python-3.
 Tips, tricks, and idioms to adopt
 =================================
 
-Exceptions
-----------
+Old Exception Syntax
+--------------------
 
-In order for code to function on Python-2.6+ and Python-3, use the
-new exception-catching syntax which uses the ``as`` keyword::
+Until Ansible-2.4, modules needed to be compatible with Python-2.4 as
+well.  Python-2.4 did not understand the new exception-catching syntax so
+we had to write a compatibility function that could work with both
+Python-2 and Python-3.  You may still see this used in some modules:
+
+.. code-block:: python
+
+    from ansible.module_utils.pycompat24 import get_exception
+    [...]
 
     try:
         a = 2/0
-    except ValueError as e:
```

Comment:
I wonder if something has been lost here, maybe this could be expanded to

A) This is how you should detail with exceptions in a way that works with Python 2.6+ & Python 3.5+
Code sample and explain why this is the correct way

B) Example of Python-2.4 only bad code

C) Example of python 2.4+ & Python 3.5+ exception handling
Note that this is only for Ansible 2.? - 2.3

D) Guidelines regarding if/when you should refactoring code from (B) or (C) to(A)